lib/fastlane/plugin/ci_cd/helper/audit_generator/audit_data_loader.rb

```diff
@@ -0,0 +1,21 @@
+# lib/fastlane/actions/helpers/audit_data_loader.rb
+require 'json'
+
+module Fastlane
+  module Helpers
```

Comment:
```suggestion
  module Helper
```

according to the directory name